integration-tests/api/medusa-config.js

```diff
@@ -24,10 +24,12 @@ module.exports = {
     redis_url: redisUrl,
     database_url: DB_URL,
     database_type: "postgres",
-    jwt_secret: "test",
-    cookie_secret: "test",
-    http_compression: {
-      enabled: enableResponseCompression,
+    http: {
+      compression: {
+        enabled: enableResponseCompression,
+      },
+      jwtSecret: "test",
+      cookieSecret: "test",
```

Comment:
suggestion: maybe we should allow to set the cookie domain and make it required when running in production to avoid being able to send the cookie everywhere else. cc @sradevski

Reply:
unless it is automatically done under the hood?

Reply:
I think it defaults to the domain from which it was set

Reply:
ok we'll need to look into this to ensure the cookie doesn't leak into 3rd party services (eg. sentry), but I'll tackle that as part of the bigger auth writeup.